src/tokens.c

```diff
@@ -402,6 +402,7 @@ const tokenDefinition_t const TOKENS_TRC20[NUM_TOKENS_TRC20] = {
     {{0x41,0x14,0x18,0x3f,0x3b,0xbc,0xa4,0xae,0x9f,0xc1,0xde,0x55,0xb9,0xbb,0xe2,0xd0,0x71,0x94,0x2d,0xc1,0xa6}, "CCT ", 6},
     {{0x41,0x83,0x42,0x95,0x92,0x1a,0x48,0x8d,0x9d,0x42,0xb4,0xb3,0x02,0x1e,0xd1,0xa3,0xc3,0x9f,0xb0,0xf0,0x3e}, "USDJ ", 18},
     {{0x41,0x18,0xfd,0x06,0x26,0xda,0xf3,0xaf,0x02,0x38,0x9a,0xef,0x3e,0xd8,0x7d,0xb9,0xc3,0x3f,0x63,0x8f,0xfa}, "JST ", 18},
+    {{0x41,0xe1,0x1c,0xdc,0x16,0x4a,0x9d,0x8c,0x1a,0xe1,0x9d,0x95,0xb0,0x16,0x52,0x78,0x69,0x0d,0x39,0xd8,0x4b}, "OKS", 18},
```

Comment:
In `parse.h`:

`#define NUM_TOKENS_TRC20 370`

When adding a new TRC20 token, this number should be increased by 1.

Reply:
Oops, fixed.

Reply:
Sorry to bother, can you please modify and set the value to 372 (instead of 371).

Because there is [another token](https://github.com/LedgerHQ/app-tron/pull/15/files) to add as well.

Thanks 🙏